Update teams-only-mode-considerations.md (#2588)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-only-mode-considerations.md

**User-specific considerations**: Some user scenarios are still evolving, and administrators may decide to temporarily postpone the upgrade of certain users while upgrading other users in the organization. We are working on addressing these scenarios; please monitor the [Office 365 Roadmap](https://www.microsoft.com/en-us/microsoft-365/roadmap) site for announcements.

> [!NOTE]

> > Meeting Room devices that don’t support Teams need to be replaced with supporting devices before moving to Teams Only mode. Alternatively get an update from the vendor on such devices.

Reviewer: ```suggestion
> Before you move to Teams Only mode you will need to replace or update devices that don’t support Teams. 
```
